### DISCUSSION TITLE
Move register_devices over to App Store Connect API

### DISCUSSION THREAD

**Discussion by andrew-muzz** [fastlane/fastlane#17407] [Ideas] [answered]
Been some really great work recently moving Match over to the App Store Connect API.
We use `register_devices` with Match so would be great if they could go through the same auth.

Have I missed something and this is supported?
or
Are there any plans to move `register_devices` over to the new API?
If this isn't in the pipeline is it something I can contribute?

**Comment by rogerluan** [comment]
Hey @andrew-muzmatch 👋 

Noted! 🤗 This has been answered here before: https://github.com/fastlane/fastlane/discussions/17395#discussioncomment-91895

There's no ongoing work over this AFAIK - any contributions are more than welcome! 🚀

**Comment by joshdholtz** [comment]
Hello!

I will be making a change for this today 😊 I meant to do it yesterday but I was feeling generally unproductive and just couldn’t get anything done 😔

**Comment by rogerluan** [comment, marked as answer]
Hey @andrew-muzmatch ! 

Josh took care of this [here](https://github.com/fastlane/fastlane/pull/17426) and this was released as part of [_fastlane_ 2.163.0](https://github.com/fastlane/fastlane/releases/tag/2.163.0) 🚀

**Comment by andrew-muzz** [comment]
Thanks @rogerluan and @joshdholtz for handling this. Just updated and works like a dream.
